lib/MusicBrainz/Server/Data/Editor.pm

```diff
@@ -70,13 +70,30 @@ sub _entity_class
     return 'MusicBrainz::Server::Entity::Editor';
 }
 
+=item lc_name
+
+Return an editor name with ASCII characters lower-cased.
```

Comment:
Another issue I ran into while adding tests...

I named the new test editor `æditorⅣ`, which broke `get_by_name` here because `lc` lowercases the roman numeral while `lower(name)` doesn't (in the C locale). So the editor couldn't be found. I wasn't actively looking for more things to break. :)